.ci/gcb-generate-diffs.yml

```diff
@@ -169,23 +169,25 @@ steps:
       args:
           - $_PR_NUMBER
 
-    - name: 'gcr.io/graphite-docker-images/terraform-google-conversion-tester'
+    - name: 'gcr.io/graphite-docker-images/downstream-builder'
       id: tgc-test
+      entrypoint: "test_tpgc.sh"
       secretEnv: ["GITHUB_TOKEN"]
       waitFor: ["diff"]
       args:
           - $_PR_NUMBER
 
-    - name: 'gcr.io/graphite-docker-images/terraform-tester'
+    - name: 'gcr.io/graphite-docker-images/downstream-builder'
```

Comment:
should this use the test_terraform.sh entrypoint?

Reply:
yep!